diff -ur a/build/targets.device.mak b/build/targets.device.mak
--- a/build/targets.device.mak	2020-09-01 20:21:31.506885598 +0200
+++ b/build/targets.device.mak	2020-09-01 20:22:08.953114454 +0200
@@ -68,5 +68,5 @@
 	mkdir -p $(BUILD_DIR)/binpack
 	cp $(BUILD_DIR)/flasher.light.bin $(BUILD_DIR)/binpack
 	cp $(BUILD_DIR)/epsilon.onboarding.internal.bin $(BUILD_DIR)/epsilon.onboarding.external.bin $(BUILD_DIR)/binpack
-	cd $(BUILD_DIR) && for binary in flasher.light.bin epsilon.onboarding.internal.bin epsilon.onboarding.external.bin; do shasum -a 256 -b binpack/$${binary} > binpack/$${binary}.sha256;done
+	cd $(BUILD_DIR) && for binary in flasher.light.bin epsilon.onboarding.internal.bin epsilon.onboarding.external.bin; do sha256sum -b binpack/$${binary} > binpack/$${binary}.sha256;done
 	cd $(BUILD_DIR) && tar cvfz binpack-$(MODEL)-`git rev-parse HEAD | head -c 7`.tgz binpack/*
